Jenkinsfile

```diff
@@ -3,6 +3,7 @@ stage("Sanity Check") {
     ws('workspace/gluon-nlp-lint') {
       checkout scm
       sh """#!/bin/bash
+      git clean -f -d -x
```

Comment:
What about adding `--exclude=<pattern> ` to `git clean` to sapare the `tests/externaldata` folder from being deleted every time?